radio/src/eeprom_raw.cpp

```diff
@@ -382,6 +382,10 @@ void eeLoadModel(uint8_t id)
     LOAD_MODEL_BITMAP();
 
     SEND_FAILSAFE_1S();
+
+#if defined(SDCARD)
```

Comment:
the ifdef is not needed, you defined it in audio_arm.h
instead I would see it also defined as nothing in audio_avr.h and beep.h